### DISCUSSION TITLE
Copilot is not working

### DISCUSSION THREAD

**Discussion by rpayanm** [community/community#58055] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

Hi there,
I have been using copilot in Phpstorm until last month, but now it is working anymore. I have an active subscription. 
PhpStorm version: 2023.1.2
Copilot version: 1.2.8.2631

When I go and click here:
![Screenshot from 2023-06-14 11-28-57](https://github.com/community/community/assets/6087454/e397f0b0-9ef6-4745-875b-df4d9407e68a)

I get this pop up:
![Screenshot from 2023-06-14 11-29-19](https://github.com/community/community/assets/6087454/b825e9b6-71c8-40db-8f07-654dc2f855c3)

And nothing happens.

I checked the log `idea.log`, and I did not see any error, only this warning when I open the IDE:
```
2023-06-14 11:38:38,179 [   6120]   WARN - #c.i.o.w.i.s.w.StatusBarWidgetsManager - com.github.copilot.statusBar.CopilotWidgetFactory doesn't define id for extension (point=com.intellij.statusBarWidgetFactory). Please specify `id` attribute.
```

Could you help me how to fix it? Please.

**Comment by werniq** [comment]
Try to update Phpstorm, copilot. 
Reinstall phpstorm, copilot. Check if that version of Copilot is compatible with Phpstorm.
I recommend you to contact support as well.

**Reply by rpayanm** [reply, marked as answer]
I did it work. I had been using a slow internet, after I changed to a better one and Copilot started to work again.

**Comment by kombatkos** [comment]
Please help with copilot for Xcode authorization. 
I don't get ok status when logging in.
- Copilot versions in the photo.
- Xcode version 13.4
- Use OpenVPN

<img width="514" alt="Снимок экрана 2023-06-20 в 14 32 32" src="https://github.com/community/community/assets/56546669/a7f7bedc-9899-4844-b9a7-43873bcc9655">

<img width="894" alt="Снимок экрана 2023-06-20 в 14 33 05" src="https://github.com/community/community/assets/56546669/906031fa-4447-4223-a631-cd347dd90457">
